src/wavesurfer.js

```diff
@@ -1391,6 +1391,7 @@ export default class WaveSurfer extends util.Observer {
      * Loads audio using Web Audio buffer backend.
      *
      * @private
+     * @emits WaveSurfer#waveformReady
```

Comment:
waveformReady is not the correct event name though?

Reply:
@thijstriemstra Sorry, it seemed to me that the convention was to use the "camel case" equivalents of the "snake case" event name strings. I will switch to snake case.